templates/000-global.cfg.askama

```diff
@@ -17,6 +18,10 @@ global
     master-worker
     mworker-max-reloads 10
     log 127.0.0.1 len 4096 local0
+{%- if let Some(addr) = syslog_endpoint %}{#+ #}
+    log {{addr}} len 4096 local0
```

Comment:
@danmcd the address ends up here. Haproxy is in charge of whatever happens next.
We already pass in CNS type hostnames into Haproxy config files and they seem to work.